Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19-alpine as builder
+FROM golang:1.22-alpine as builder
```

Comment:
This is then going to be even harder to merge as updating this is [more involved](https://github.com/brave-intl/bat-go/pull/2251) than simply bumping it here.

Is there any particular reason this is needed?

Reply:
There are standard library features (like `slices.Contains`) in 1.21 that we'd like to use. I'm aware of the implications but since we're planning to move this to its own repository I think it's ok.

Reply:
I checked – the newly added modules require `1.20`. Do please leave this intact.

Reply:
@Sneagan You can have it for free as we already have in SKUs – https://github.com/brave-intl/bat-go/blob/master/services/skus/model/model.go#L632-L656.